README.md

```diff
@@ -1,2 +1,95 @@
+structure will look something like this (truncated for readability):
+
+```
+must-gather.local.7079684121838056125/
+└── XXXXXXX-must-gather-sha256-4f688c549604a1ca3b0afdfe5adc26d525a1ba6a16c23b7a7445212a3839e1c2
+    ├── apiservices
+    ├── cluster-scoped-resources
+    │   ├── apiextensions.k8s.io
+    │   │   └── customresourcedefinitions
+    │   ├── metal3.io
+    │   │   └── provisionings
+    │   ├── nmstate.io
+    │   │   ├── nodenetworkconfigurationenactments
+    │   │   ├── nodenetworkconfigurationpolicies
+    │   │   └── nodenetworkstates
+    ├── namespaces
+    │   ├── openshift-cnv
+    │   ├── openshift-machine-api
+    │   ├── openshift-marketplace
+    │   ├── openshift-operator-lifecycle-manager
+    │   ├── openshift-sriov-network-operator
+    │   └── openstack
+    │       ├── cdi.kubevirt.io
+    │       │   └── datavolumes
+    │       ├── controller-cloudinit
+    │       ├── controller-controller-0-networkdata
+    │       ├── core
+    │       │   ├── configmaps
+    │       │   ├── pods
+    │       │   └── secrets
+    │       ├── crs
+    │       │   ├── openstackbaremetalsets.osp-director.openstack.org
+    │       │   ├── openstackclients.osp-director.openstack.org
+    │       │   ├── openstackcontrolplanes.osp-director.openstack.org
+    │       │   ├── openstackipsets.osp-director.openstack.org
+    │       │   ├── openstacknets.osp-director.openstack.org
+    │       │   ├── openstackplaybookgenerators.osp-director.openstack.org
+    │       │   ├── openstackprovisionservers.osp-director.openstack.org
+    │       │   └── openstackvmsets.osp-director.openstack.org
+    │       ├── hco.kubevirt.io
+    │       ├── heat-env-config
+    │       ├── image.openshift.io
+    │       ├── k8s.cni.cncf.io
+    │       │   └── network-attachment-definitions
+    │       ├── kubevirt.io
+    │       │   ├── virtualmachineinstances
+    │       │   └── virtualmachines
+    │       ├── openstackclient
+    │       ├── openstackclient-sh
+    │       ├── openstack-f7898fddb-jhvwk
+    │       ├── openstack-httpd-config
+    │       ├── openstackplaybook-script-default
+    │       ├── operators.coreos.com
+    │       │   ├── catalogsources
+    │       │   └── clusterserviceversions
+    │       ├── osp-controlplane-ssh-keys
+    │       ├── osp-director-operator-controller-manager-6f4d44bf6-qmjhd
+    │       ├── osp-director-operator-index-nlrvj
+    │       ├── osp-director-operator-openstackplayboo-4750bfb2-dockercfg-xhvmr
+    │       ├── pods
+    │       │   ├── openstackclient
+    │       │   ├── openstack-f7898fddb-jhvwk
+    │       │   ├── osp-director-operator-controller-manager-6f4d44bf6-qmjhd
+    │       │   ├── osp-director-operator-index-nlrvj
+    │       │   └── virt-launcher-controller-0-7tvhg
+    │       ├── snapshot.kubevirt.io
+    │       ├── tripleo-deploy-config
+    │       ├── tripleo-passwords
```

Comment:
we might actually want to exclude password files from the tarball

Reply:
Ack.  Added crude grep to exclude secrets with `password` in the name.